razorpay-payments.php

```diff
@@ -527,14 +572,19 @@ function check_razorpay_response()
             {
                 $success = false;
                 $error = 'Customer cancelled the payment';
-
                 $this->handleErrorCase($order);
             }
 
             $this->updateOrder($order, $success, $error, $razorpayPaymentId);
 
+            $this->redirectUser($order);
+        }
+
+        protected function redirectUser($order)
+        {
```

Comment:
@captn3m0 This should be good to go unless the `$razorpayPaymentId` isn't set. Do we handle that case too? If the order is already marked paid by the webhook, then surely the $razorpayPaymentId should be saved right?